tools/cep_matcher.md

```diff
@@ -0,0 +1,90 @@
+CEP Matcher
+===========
+
+The CEP Matcher tab lets you easily view some current pings of any ping type. To access it, follow 
+[these first few directions](cookbooks/realtime_analysis_plugin.md) for accessing the CEP. Once there,
+click on the "Matcher" tab. The message-matcher is set by default to `TRUE`, meaning all pings will
+be matched. Click "Run Matcher" and a few pings will show up.
+
+## Editing the Message Matcher
+
+Changing the message matcher will filter down the accepted pings, letting you hone in on a certain type.
```

Comment:
For whatever reason, the dictionary doesn't think "matcher" is a word. Please add it to the `.spelling` file.